File: IS (Java)/XOR_AND.java
public class XOR_AND {
    public static void main(String[] args) {
        String s = "Hello\nWorld";

        for (char c : s.toCharArray())
            System.out.print((char)(c ^ 127));

        System.out.println();

        for (char c : s.toCharArray())
            System.out.print((char)(c & 127));
    }
}

File: IS (C++)/XOR_AND.cpp
#include <iostream>
#include <string>

using namespace std;

int main() {
    string s = "Hello\nWorld";

    for (char c : s)
        cout << (char)(c ^ 127);
    cout << endl;

    for (char c : s)
        cout << (char)(c & 127);
    cout << endl;

    return 0;
}
